editorSupport/sublime-reason/Reason.py

```diff
@@ -28,7 +28,6 @@ def reason_command_line(file_name):
 
     return [
         refmt,
-        '--use-stdin', 'true',
         '--parse', 're',
         '--interface', is_interface(file_name),
```

Comment:
I need to fix it here since it exhibits the same situation as the vim extension above.